src/containers/WatchList/DeleteMovie/container.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Modal } from 'antd'
+import { connect } from 'react-redux'
+import { openNotification } from 'Utils'
```

Comment:
this import is related to the our modules, not 3rd party

Reply:
✅